docs/painless/painless-operators-boolean.asciidoc

```diff
@@ -0,0 +1,516 @@
+[[painless-operators-boolean]]
+=== Operators: Boolean
+
+[[boolean-not-operator]]
+==== Boolean Not
+
+Boolean not will flip a boolean value from true to false or false to true using the bang operator.  The format is a bang operator followed by an expression.
+
```

Comment:
I think I'd put `not` in code font or all caps to make it a bit easier to read correctly. I'd probably say: Boolean `not` flips a boolean value between `true` and `false` and is represented by the ! (bang) operator. The format is the bang operator followed by an expression.

Reply:
Used caps, tried to do this consistently for all operators that it made sense for.